pandas/core/arrays/categorical.py

```diff
@@ -2650,11 +2650,8 @@ def _convert_to_list_like(list_like):
         return list_like
     if is_sequence(list_like) or isinstance(list_like, tuple) or is_iterator(list_like):
         return list(list_like)
-    elif is_scalar(list_like):
-        return [list_like]
-    else:
-        # TODO: is this reached?
-        return [list_like]
+
+    return [list_like]
```

Comment:
any idea why we're not doing `if is_list_like(x): return x; return [x]`?

Reply:
Not sure, that's cleaner and will try

Reply:
Looks like a lot of things fail if we try that (e.g., tuples need to be converted to lists I think)